script/install.sh

```diff
@@ -406,19 +406,28 @@ install_dlang_installer() {
     tmp=$(mkdtemp)
     local mirrors=(
         "https://dlang.org/install.sh"
-        "https://nightlies.dlang.org/install.sh"
+        "http://downloads.dlang.org/other/install.sh"
     )
-    local keyring_mirrors=(
+    local keyring_mirrors_bootstrap=(
         "https://dlang.org/d-keyring.gpg"
-        "https://nightlies.dlang.org/d-keyring.gpg"
+        # use fixed commit hash to protect against tampering of the repo when we
+        # don't yet have an old keyring to verify the new one
+        "https://github.com/dlang/dlang.org/raw/7dfae2f8a4c33a2471767cd5c2e45a5e7da4ef10/d-keyring.gpg"
+    )
+    local keyring_mirrors_verify=(
+        "https://dlang.org/d-keyring.gpg"
+        # use possibly insecure but up-to-date mirrors when we have an old
+        # keyring to verify the download
+        "http://downloads.dlang.org/other/d-keyring.gpg"
+        "https://github.com/dlang/dlang.org/raw/master/d-keyring.gpg"
```

Comment:
We haven't uploaded a signature of the current keyring from a previous one into GitHub, so this would always fail, no?

Reply:
Here is the signature https://github.com/dlang/dlang.org/commit/7dfae2f8a4c33a2471767cd5c2e45a5e7da4ef10#diff-bcd49c888a108fa6e5ad8b14270bafd5.